job-service/pom.xml

```diff
@@ -97,6 +97,11 @@
             <groupId>com.github.jobservice</groupId>
             <artifactId>job-service-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is this up here instead of down with the other `junit-jupiter` dependency that you're adding?

Reply:
sorry, moved to the right place with the other test dependencies